src/components/Categories/MonthlyGoodModule.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+	<div class="tw-relative">
+		<kv-responsive-image
+			class="image-sizing"
+			:images="monthlyGoodImages"
+			loading="lazy"
+			:alt="'Monthly Good'"
```

Comment:
Do we need the extra single quotes?

Reply:
Or actually it doesn't seem to work without them, nevermind. :)

Reply:
Oh right, you can change to `alt="Monthly Good"`